tests/functional/test_hooks/test_wx_lib_pubsub.py

```diff
@@ -26,6 +39,7 @@ def test_wx_lib_pubsub_protocol_default(pyi_builder):
     pyi_builder.test_script('pyi_hooks/wx_lib_pubsub.py')
 
 @xfail_py3
+@xfail(wxPython_fail, reason='Unsupported wxPython version')
```

Comment:
Unsupported by PyInstaller. If `test_import_respects_path` can be made to pass, then this will pass too. The root cause is the the `FrozenImporter` first searches `__path__` in the PYZ, then `__path__` in the filesystem is searched. Instead, the order should be like this pseudo-code: `for x in __path__: search(x, PYZ); search(x, local_fs);`.